hera_sim/utils.py

```diff
@@ -552,23 +552,25 @@ def reshape_vis(
         Input data reshaped to desired shape.
     """
     if invert:
-        out = np.zeros((ant_1_array.size, 1, n_freqs, n_pols), dtype=complex)
+        out = np.zeros((ant_1_array.size, n_freqs, n_pols), dtype=complex)
     else:
         out = np.zeros((n_times, n_freqs, 2 * n_ants, 2 * n_ants), dtype=complex)
 
     # If we have numba, then this is a bit faster.
     if HAVE_NUMBA and use_numba:
-        return jit_reshape_vis(
```

Comment:
Maybe just put add `# pragma: nocover` after `if HAVE_NUMBA and use_numba:` since coverage seems to be missing this. (I've found that coverage generally has a hard time with anything that has been jitted, although it's weird that it doesn't register calling these functions.)